### ISSUE TITLE
iOS 11 problems

### ISSUE THREAD

**Issue by xale76** [pushandplay/cordova-plugin-apprate#178]
It seems that on iOS11 when user clicks on "rate" anything happens.
What's the problem?

this is my code (JS phonegap cordova)

AppRate.preferences = {
  usesUntilPrompt: 1,
  promptAgainForEachNewVersion: false,
  inAppReview: true,
  simpleMode: true,
  storeAppURL: {
      ios: 'xxxxxxx',
      android: 'xxxxxxx' 
       },
  customLocale: {
    title: 'my tytle',
    message: 'my message',
    cancelButtonLabel: 'cancel',
    laterButtonLabel: 'later',
    rateButtonLabel: 'rate'
     
   },
 
  callbacks: {
   
    onButtonClicked: function(buttonIndex){
      setAskVal( buttonIndex);
    }
  }
  };

  AppRate.promptForRating();

**Comment by viking2917** [comment]
I also could not get this to work on IOS 11 - see https://github.com/pushandplay/cordova-plugin-apprate/issues/182 for details. 

By doing 
`inAppReview: false`
my app was able to leave reviews, but it takes the user to the app store which is more work for them and less desirable.

**Comment by jmcrthrs** [comment]
https://github.com/pushandplay/cordova-plugin-apprate/issues/182#issuecomment-338582900

**Comment by viking2917** [comment]
yes - per @jmcrthrs link, it does in fact work on iOS 11 for me - the issue is that the submit button is greyed out when you have a development app, but after it's released in the app store, it works. Wildly counter-intuitive, but works.
